dev-docs/bidders/optidigital.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: bidder
+title: Optidigital
+description: Prebid Optidigital Bidder Adapter
+biddercode: optidigital
+pbjs: true
+floors_supported: true
+tcf2_supported: true
+usp_supported: true
+schain_supported: true
+media_types: banner
+gvl_id: 915
+sidebarType: 1
+---
+
+### Note:
+
+The Optidigital Bidding adapter requires setup before beginning. Please contact us at prebid@optidigital.com
+
+### Bid Params
```

Comment:
It's always nice to an an example configuration of an ad unit here. If you happen to have a test `publisherId` and `placementId`, this makes integration a lot faster and smoother

Reply:
The example of an ad unit configuration with test placementId and publisherId was added in the last commit.